pysb/tests/test_util.py

```diff
@@ -0,0 +1,41 @@
+from pysb.core import *
+from pysb.util import *
+from pysb.examples import earm_1_0
+
+
+def test_alias_model_components():
+    """
+    Tests that alias_model_components() exports to the global namespace
+    """
+    m = Model(_export=False)
+    m.add_component(Monomer('temp_monomer_A', _export=False))
+    try:
+        temp_monomer_A
+        raise NameError('Monomer A should not be defined in global namespace')
+    except NameError:
+        # We expect this exception due to _export=False
+        pass
+    alias_model_components(m)
+
+    # Use global version of temp_monomer_A (needed to delete it)
+    global temp_monomer_A
+
+    # A should now be defined in the namespace - try deleting it
+    assert isinstance(temp_monomer_A, Monomer)
+
+    # Delete the monomer to cleanup the global namespace
+    del temp_monomer_A
+
+
+def test_rules_using_parameter():
+    """
+    Tests for rules_using_parameter() in pysb.util
+    """
+    # Get rules matching parameter name
+    r = rules_using_parameter(earm_1_0.model, 'kc1')
```

Comment:
This probably shouldn't depend on earm_1_0 but rather define its own little model to test. I wrote a decorator `pysb.testing.with_model` which lets you just start declaring components in a test function without declaring the model or worrying about cleaning up at the end, so writing mini-models in test functions is not so painful. See the tests in test_core for many examples.

The variable name `r` is not great, as the return value is a ComponentSet not a Rule. `rules` or `components` would be better.